### DISCUSSION TITLE
GetX Roadmap

### DISCUSSION THREAD

**Discussion by dasaintgray** [jonataslaw/getx#2794] [General]
Is there any roadmap for this project? It's a pretty long-lost update. are you planning to give up on this? Thanks

**Comment by ilgnefz** [comment]
You can use the preview version [get 5.0.0-release-candidate-4 ](https://pub.dev/packages/get/versions/5.0.0-release-candidate-4).

**Reply by dasaintgray** [reply]
I thought, the maintainer of this project already give up because of the negative comments.
